wiki/osu!academy/id.md

```diff
@@ -1,17 +1,21 @@
+---
+needs_cleanup: true
```

Comment:
@cl8n do translations also get this too?

Reply:
content/meaning needs work => add to en
the translation itself needs work, but en is fine => add here

Reply:
the translation copied en, but en needs work => add to en only?

Reply:
ya